Hunting Queries/MultipleDataSources/RareDNSLookupWithDataTransfer.yaml

```diff
@@ -82,19 +82,9 @@ query: |
   | extend IPCustomEntity = DestinationIP
   | sort by sum_SentBytes desc
   ),
-  (WireData
-  | where TimeGenerated between(starttime..endtime)
-  | where Direction == "Outbound"
-  | where ipv4_is_private(RemoteIP) == false
-  | project DataType = Type, TimeGenerated , SourceIP = LocalIP , SourcePort = LocalPortNumber , DestinationIP = RemoteIP, DestinationPort = RemotePortNumber, ReceivedBytes, SentBytes
-  | summarize sum(ReceivedBytes), sum(SentBytes), ConnectionCount = count() by DataType, SourceIP, SourcePort, DestinationIP, DestinationPort
-  | extend IPCustomEntity = DestinationIP
-  | extend DataType = Type
-  | sort by sum_SentBytes desc
-  ),
   (VMConnection
   | where TimeGenerated between(starttime..endtime)
-  | where Direction == "outbound"
+  | where Direction == "Outbound"
```

Comment:
@v-vdixit Please also use '=~' instead of '=='. former is used for case-insensitivity